packages/apps-config/package.json

```diff
@@ -25,7 +25,7 @@
     "@sora-substrate/type-definitions": "1.3.34",
     "@subsocial/types": "0.5.9-dev.2",
     "@zeitgeistpm/type-defs": "0.2.0",
-    "moonbeam-types-bundle": "1.1.25",
+    "moonbeam-types-bundle": "latest",
```

Comment:
We actually want this on a specific version - we had some issues with people using the `@polkadot/apps-config` package (in other projects) running into issues here.

I know it is not perfect, but at the same time it does open up gaps.